.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 pnpm-lock.yaml
+package-lock.json
 .nyc_output
-coverage
\ No newline at end of file
+coveragep
```

Comment:
What's this?

Reply:
Hmm, typo?
There was some coverage thing in the repo?

Reply:
Okay, just clarifying that it's not intentional